modules/olm-deprecations-schema.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding/olm-packaging-format.adoc
+
+[id="olm-deprecations-schema_{context}"]
+= olm.deprecations schema
+
+The optional `olm.deprecations` schema defines deprecation information for packages, bundles, and channels in a catalog. Operator authors can use this schema to provide relevant messages about their Operators, such as support status and recommended upgrade paths, to users running those Operators from a catalog.
+
+An `olm.deprecations` schema entry can contain at least one of the following `reference` types, which indicates the deprecation scope. After the Operator is installed, any deprecation messages can be viewed as status conditions on the related `Subscription` object.
+
+.Deprecation `reference` types
+[cols="1,2,1",options="header"]
+|===
+|Type |Scope |Status condition
+
+|`olm.package`
+|Represents the entire package
+|`PackageDeprecated`
+
+|`olm.channel`
+|Represents one channel
+|`ChannelDeprecated`
+
+|`olm.bundle`
+|Represents one bundle version
+|`BundleDeprecated`
+
+|===
+
+Each `reference` type has their own requirements, as detailed in the following example.
+
+[WARNING]
+====
+If an `olm.deprecations` schema is determined invalid by the `opm validate` command, the entire catalog is invalid.
```

Comment:
I'd have to look for the citation, but in general we should be saying something like "if any part of FBC is invalid, then all the FBC is considered invalid" and the deprecations stuff is just add'l FBC schema.

Reply:
We dropped this whole warning in the upstream, as immaterial since FBC is either valid/invalid with no middle-ground.